### ISSUE TITLE
Using Outlined Edit Text from Material Design for Login Screen

### ISSUE THREAD

**Issue by madhurgupta10** [commons-app/apps-android-commons#2864]
**Summary:** 
We use the new Outlined Edit Text from Material Design which is used by Gmail. It looks super cool :)

link - https://material.io/design/components/text-fields.html#usage

**Screen-shots:** 
![image](https://user-images.githubusercontent.com/30932899/55630484-68d4c000-57d3-11e9-8cc9-88cd74618641.png)

**Would you like to work on the issue?**
Yes

**Comment by maskaravivek** [comment]
I don't think that just updating the login screen with this UI would be a good idea.

**Comment by madhurgupta10** [comment]
@maskaravivek we can replace all edit text with it, so that the UI is unified across the app.

**Comment by madhurgupta10** [comment]
@misaochan @maskaravivek The Outlined Edit Text has become very popular now, and it is complimentary with the Dark Theme.

![Screenshot_20200313-233752](https://user-images.githubusercontent.com/30932899/76664388-7d453f00-6584-11ea-84c5-b983d9085924.png)

**Comment by misaochan** [comment]
I'm not sure how it would work with our upload text fields, to be honest... but we could provide input if you can share a mockup of how you think it could look. :)
